modules/images/auto-sizes/load.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Module Name: Auto Sizes for Lazy-loaded Images.
+ * Description: Implements auto sizes for lazy loaded images.
```

Comment:
Does `module-i18n.php` need to be updated to include these strings?

Reply:
Good question. I'm not sure if that is something that needs to happen as part of a new module PR or is automated as part of the release process.

Reply:
No need to update, this is part of the release process (see https://make.wordpress.org/performance/handbook/performance-lab/releasing-the-plugin/#update-translation-strings).